packages/frontend/src/services/IndexerCache.js

```diff
@@ -0,0 +1,144 @@
+import Cache from '../utils/cache';
+
+export class IndexerCache extends Cache {
```

Comment:
Lets rename this file to indexer-cache to avoid mixing CamelCase and kebab-case.